chia/simulator/block_tools.py

```diff
@@ -1073,7 +1079,13 @@ def get_consecutive_blocks(
             blocks_added_this_sub_slot = 0  # Sub slot ended, overflows are in next sub slot
 
             # Handle overflows: No overflows on new epoch or sub-epoch
-            if new_sub_slot_iters is None and num_empty_slots_added >= skip_slots and not pending_ses:
+
+            if (
+                new_sub_slot_iters is None
+                and num_empty_slots_added >= skip_slots
+                and not pending_ses
+                and not skip_overflow
+            ):
                 for signage_point_index in range(
```

Comment:
should we check `min_signage_point` here too? probably not relevant for this PR tests, but maybe for someone who wants to use this new feature for overflow blocks as well?

Reply:
yes ill add it that would also work for overflows

Reply:
@fchirica please check if this review comment is still relevant, and resolve it if not. Open threads block merge.